nsxt/resource_nsxt_policy_host_transport_node_collection.go

```diff
@@ -256,6 +270,11 @@ func resourceNsxtPolicyHostTransportNodeCollectionRead(d *schema.ResourceData, m
 		d.Set("sub_cluster_config", sccList)
 	}
 	d.Set("transport_node_profile_path", obj.TransportNodeProfileId)
+	if util.NsxVersionHigherOrEqual("9.1.0") {
+		if len(d.Get("network_span_paths").([]interface{})) > 0 {
```

Comment:
I don't think we need the null check here, since it will prevent nullifying paths if those were deleted on backend

Reply:
I actually do not understand what @ksamoray meant do here... it seems we are setting network_span_paths only if we already have some paths in the schema?